libr/anal/p/anal_arm_cs.c

```diff
@@ -2439,7 +2439,7 @@ static int analop_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		r_strbuf_appendf (&op->esil, "%s,!,?{,32,%s,=,BREAK,},0,%s,=,%s,%s,<<,0x80000000,&,!,?{,1,%s,+=,11,GOTO,}", REG (1), REG (0), REG (0), REG (0), REG (1), REG (0));
 		break;
 	case ARM_INS_IT:
-		r_strbuf_appendf (&op->esil, "2,$$,+,pc,=");
+		r_strbuf_appendf (&op->esil, "2,pc,+,%d,pc,-,pc,:=", op->size);
```

Comment:
i dont think pc is the same as $$

Reply:
yes, I found that out, pc is $$ + op->size.

Reply:
the register is updated before stepping

Reply:
But that behaviour changes depending on thumb, arm and the alignment if its 2 or 4. We have tests for this madness and thats why it was using $$

Reply:
Because you cant easily compute that from pc

Reply:
why is this behavior changed based on thumb? anyways, I need to deprecate $$ for esil_dfg

Reply:
op->size is right regardless of whether its thumb right?

Reply:
I think that we had enough tests to confirm that the behaviour was correct. We can do a blind guess and merge and hopefully we didnt missed any test. Sgty?